webui/react/src/pages/ResourcePool/ResourcepoolDetail.tsx

```diff
@@ -277,7 +316,20 @@ const ResourcepoolDetailInner: React.FC = () => {
             size={ShirtSize.Large}
           />
         </Section>
-        {!!topologyAgentPool.length && poolname && <Topology nodes={topologyAgentPool} />}
+        {agents.length > MAX_USABLE_NODES || totalSlots > MAX_USABLE_SLOTS ? (
+          <Section title="Topology">
+            <div className={css.warningContainer}>
+              <p>
+                Current Topology view is disabled for pools with more than {MAX_USABLE_NODES} nodes
```

Comment:
HK was suggesting we do not show it instead of a msg here

Reply:
ok when condition is matched, nothing shows up